client/functions.lua

```diff
@@ -382,6 +382,29 @@ function QBCore.Functions.SpawnVehicle(model, cb, coords, isnetworked, teleportI
     if cb then cb(veh) end
 end
 
+function QBCore.Functions.SpawnNetworkedVehicle(model, cb, coords, heading, teleportInto)
```

Comment:
This is unnecessary as QBCore.Functions.SpawnVehicle already has a networked option

Reply:
Hi Berkie,

Last I checked the networked flag wasn't causing persistency for vehicles in between player sessions. Has this changed?

E.g. Spawning a vehicle via the server will give control of that vehicle to the server if no player is within the scope of that entity's existency. Meaning that vehicle will stay in the server even if the client that called for the creation of vehicle disconnects.

I'll admit it's been a few months since I have doing any coding for FiveM (or since this PR was created). Has this changed?

Reply:
Yes it has, that issue has been fixed

Reply:
Okay, cool.

This function has been removed. Thanks for the info 👍🏽